apps/api/src/app/change/usecases/create-change.usecase.ts

```diff
@@ -0,0 +1,41 @@
+import { Injectable } from '@nestjs/common';
+import { ChangeRepository } from '@novu/dal';
+import { diff, diffApply } from '../utils';
+import { CreateChangeCommand } from './create-change.command';
+
+@Injectable()
+export class CreateChange {
+  constructor(private changeRepository: ChangeRepository) {}
+
+  async execute(command: CreateChangeCommand) {
+    const res = await this.changeRepository.find(
```

Comment:
```suggestion
    const changes = await this.changeRepository.find(
```

Maybe something like this to better denote what the variable does? I think it's better to avoid ambiguous variables' name for easier read down the road. What do you think?

Reply:
Another small idea, we can move the heaviylifting of this query to inside the repository, and have a custom method there called: 
```
getEntityChanges(entityType: ChangeEntityType, entityId: string): Promise<ChangesEntity[]>
```

Wdyt?